### PR TITLE
Added error icon for failed fetching of fee rates

### PR DESCRIPTION
**Pull request by shubhamkmr04**
I have added the error icon on `EditFee.tsx` in case our API does not return fee rates
Screenshot from 2021-09-01 13-51-00

### CODE REVIEW

**Comment by shubhamkmr04** [comment]
@kaloudis please check this out.

File: images/SVG/Error Circle.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="120" height="120" viewBox="0 0 120 120" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="60" cy="60" r="56" stroke="#E14C4C" stroke-width="8"/>
```
Comment (kaloudis) [inline images/SVG/Error Circle.svg:2]: We should be able to merge these two SVGs: take this line and put it on line two of the other file

**Comment by kaloudis** [comment]
Looks good. Let's just try to make it a single SVG

**Comment by kaloudis** [comment]
@Bosch-0 do we have a combined icon for this yet?

**Comment by Bosch-0** [comment]
Should be able to export the icon from here: https://www.figma.com/file/TqAM4LvuqvY7FBzFoaezJC/Zeus?node-id=3623%3A18354

Will DM you the SVG export too if that's easier than going into Figma

**Comment by shubhamkmr04** [comment]
Just fixed error icon by using only one svg file.

**Comment by shubhamkmr04** [comment]
@kaloudis please checkout if the PR is ready to merge.

File: images/SVG/Error Cross.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="62" height="62" viewBox="0 0 62 62" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M4 4L57.9999 58" stroke="#E14C4C" stroke-width="8" stroke-linecap="round"/>
+<path d="M58 4L4.00005 58" stroke="#E14C4C" stroke-width="8" stroke-linecap="round"/>
+</svg>
```
Comment (kaloudis) [inline images/SVG/Error Cross.svg:4]: this file can be deleted now

File: images/SVG/Error Circle.svg (hunk 1 above)
Comment (kaloudis) [reply images/SVG/Error Circle.svg:2]: this file can be deleted now

File: views/EditFee.tsx [hunk 3]
```diff
@@ -274,16 +275,30 @@ export default class EditFee extends React.Component<NodeInfoProps, SendState> {
...
+                            style={{
+                                flex: 1,
+                                alignItems: 'center',
+                                justifyContent: 'center',
+                                top: 80
+                            }}
+                        >
+                            <View>
+                                <ErrorIcon/>
+                            </View>
```
Comment (kaloudis) [inline views/EditFee.tsx:289]: Try to remove the surrounding `<View>` tag from this - I don't think it's necessary

**Review by kaloudis — CHANGES_REQUESTED** [review]

**Comment by shubhamkmr04** [comment]
@kaloudis Done!

**Review by kaloudis — APPROVED** [review]

**Pull request merged**